recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2020.02') }}
```

Comment:
OGDF builds a static library by default, for which the `run_exports` doesn't make much sense then.
However, if it's possible (it's been ages since I last used OGDF so can't really remember) please try to build it as a shared lib.

Reply:
Indeed ogdf is built as a static lib when looking at the installed files: `$PREFIX/lib/libOGDF.a`. Not sure how to build it as a shared library without modifying the source repo thought.

@scopatz can you confirm it's ok to remove the `run_exports` since the lib is static?

Reply:
Maybe adding `-DBUILD_SHARED_LIBS=ON` to the `cmake` arguments would work?
ref: https://github.com/ogdf/ogdf/blob/catalpa-202002/cmake/ogdf.cmake#L4

Reply:
I have removed it. Please comment if it must be reverted.

Reply:
`DBUILD_SHARED_LIBS` did the trick indeed.